architecture/additional_concepts/authentication.adoc

```diff
@@ -195,8 +195,8 @@ xref:../../admin_guide/service_accounts.adoc#admin-guide-service-accounts[servic
 account] can be used as a constrained form of OAuth client. Service accounts can
 only request a subset of
 xref:../../admin_guide/scoped_tokens.adoc#admin-guide-scoped-tokens[scopes] that
-allow access to some basic user information and role-based power inside of the
-service account's own namespace:
+allow access to some basic user information, API server authentication troubleshooting events,  
```

Comment:
Please remove this change.  This section is about scopes, debugging has nothing to do with that.  Otherwise LGTM.